package.json

```diff
@@ -10,6 +10,11 @@
     "pretty-quick": "~3.1.0"
   },
   "scripts": {
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "npm run bootstrap",
+    "bootstrap": "lerna bootstrap",
+    "build": "lerna run build",
+    "build.tests": "lerna run --scope '{@astrouxds/astro-web-components,@astrouxds/react}' build",
+    "test": "lerna run --parallel --ignore @astrouxds/angular test"
```

Comment:
@micahjones13 Your previous changes worked great, so feel free to drop my changes. I mainly just moved that `start-server-and-test` command into the `web-components` package so the top level `package.json` could be as abstract as possible. 
It's approved by me, with or without my commits

Reply:
I was trying to figure out a way to abstract the start-server-and-test and didn't even think about just moving it to web-components 😆 Thanks!